code/modules/clothing/head/jobs.dm

```diff
@@ -26,7 +32,7 @@
 	return(FIRELOSS)
 
 /obj/item/clothing/head/chefhat/relaymove(mob/user, direction)
-	if(!istype(user, /mob/living/simple_animal/mouse) || !isliving(loc) || !prob(20))
+	if(!istype(user, /mob/living/simple_animal/mouse) || !isliving(loc) || !prob(mousecontrolprobability))
 		return
 	var/mob/living/L = loc
 	step_towards(L, get_step(L, direction))
```

Comment:
might need a incapacitated check here for L, unless step already handles that

Reply:
eh, sure, why not

worse case scenario, it's just a redundant check

Reply:
eh, worst case scenario, we have a mouse puppeting a guy's unconscious body/sleeping corpse, right? then it'd be like that wallace and gromit movie with the trousers.

Reply:
dead guys too.

Reply:
Is this a means to break out of a pull while cuffed?

Reply:
Yes.